### ISSUE TITLE
[l10n] String extraction duplicates shared strings within addons

### ISSUE THREAD

**Issue by flodolo** [mozilla-mobile/mozilla-vpn-client#10088]
@mcleinman 

Unfortunately the string extraction is not working as it should: shared strings (like `vpn.commonStrings.generalUpdateBulletIntro`), are extracted twice, once in `en/addons/strings.xliff`, once in the addon (e.g. `en/addons/message_update_v2.25/strings.xliff`).

Should we ignore strings starting with `vpn.commonStrings` when generating addons? While it could be done in the l10n repository as an additional workflow step, I think the clean approach is to fix it in this repo, where addon strings are extracted.

In order to unblock translation, should I go ahead and manually remove shared strings from `whats_new_v2.25`?

**Comment by mcleinman** [comment]
Yes, we can manually remove shared strings from `whats_new_v2.25` and `message_update_v2.25`, or just close that current PR.

[I have a fix for this up](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/10091), and the description includes more diagnosis / explanation. Sorry about this.
